machine-runner/src/design/protocol.ts

```diff
@@ -38,7 +39,7 @@ export namespace SwarmProtocol {
   /**
    * Construct a SwarmProtocol
    * @param swarmName - The name of the swarm protocol
-   * @param tags - the tags used to mark the events passed to Actyx
+   * @param tagString - the tags used to mark the events passed to Actyx
```

Comment:
```suggestion
   * @param tagString - the base tag used to mark the events passed to Actyx
```

Reply:
Thanks. I missed out all the example jsdocs too.